tests/integration/test_application_variables.py

```diff
@@ -16,18 +16,32 @@ def tearDown(self):
         settings.EQ_GOOGLE_TAG_MANAGER_AUTH = None
 
     def test_google_analytics_code_and_credentials_are_present(self):
-        self.launchSurvey("test_textfield")
+        self.launchSurvey("test_feedback")
```

Comment:
Any reason for the schema change?

Reply:
Yeah, the feedback schema has a form type, the other one doesn't or the other way round.